src/sites/amsterdam/components/shared/FormNavigation.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, Icon } from '@amsterdam/design-system-react'
+import { Link } from '../../../../shared/components/Link'
+import { ChevronLeftIcon } from '@amsterdam/design-system-react-icons'
+
+const FormNavigation = ({
```

Comment:
Both the `amsterdam` and `signalen` site now have a shared `FormNavigation` component. This one also contains a class name belonging to the other site. Can we make it even more generic and move it into `src/shared/components`?